### ISSUE TITLE
binding error ,  Android Studio 3.1.1  gradle 4.1

### ISSUE THREAD

**Issue by leirace** [parthdave93/RecyclerView-States-Databinding#1]
data binding error ****msg:Cannot find the setter for attribute 'app:provider' with parameter type com.parthdave.databindingandrecyclerview.bindings.ListBindings.UserListingBinding on com.parthdave.recyclerviewbindinglibrary.databinding.CommonRecyclerviewLayoutBinding. file:/RecyclerView-States-Databinding-master/app/src/main/res/layout/activity_main.xml loc:45:28 - 45:41 ****\ data binding error ****

**Comment by parthdave93** [comment]
yes, there is some issue with android studio 3.1 will look into it this weekend and let you know. until then try if you can fix it let me know.

**Comment by leirace** [comment]
tks, I try a lot of ways. and success 

upgrade to  databinding V2, gradle-4.4
add line gradle.properties
`android.databinding.enableV2=true`
and upgrade 
me.tatarka.bindingcollectionadapter2:bindingcollectionadapter:2.3.0-beta1

Maybe some others

it's running.  I can't explain it ,  I'm newer learn databinding. 
thanks you!  it's a great sample for me.

**Comment by parthdave93** [comment]
Will update repository for others , and databinding is pretty easy
If you want help let me know.
And I hope this repo helped you in someway.
